### DISCUSSION TITLE
RFC 9250

### DISCUSSION THREAD

**Discussion by Mile-Lile** [pymumu/smartdns#983] [General]
Can you reconsider implementing DoQ? Seems it became a [standard](https://datatracker.ietf.org/doc/rfc9250/)
I know there is no big difference between DoT, DoH and DoQ for everyday smartdns user but I think there should be choice... Every major DNS provider I tested supports DoQ... CloduFlare, NextDNS, AdGuard... i read that there were some problems with go-lang libraries for DoQ but will be solved soon...
![2](https://user-images.githubusercontent.com/12752816/178450403-91db0339-d90c-4db0-9ccf-e8af2e53fd2c.JPG)
![1](https://user-images.githubusercontent.com/12752816/178450442-6dae0b98-54ed-4c2c-b308-1ea2aeaf8225.JPG)

**Comment by pymumu** [comment]
There is no plan to support it yet, because the implementation of the quic protocol is very complicated. At present, openssl is already developing the support of the quic protocol. This feature will be added when openssl supports it.

**Comment by Mile-Lile** [comment]
Fair enough, I really appreciate what you already did... I use SmartDNS
within ddwrt and I like it very much...
